### ISSUE TITLE
Relation.query() with a where not working

### ISSUE THREAD

**Issue by flessard** [ParsePlatform/parse-server#344]
If you perform this 
```
var relation = u.relation('onOtherCompany');
var query = relation.query();
query.equalTo('objectId', company.id);
```
The result is not filtered. All objects in relation is return.

**Comment by joanmarcs** [comment]
Same problem...

**Comment by gfosco** [comment]
I believe this to be fixed with #769.  Do let us know if this is still a problem!  :+1:
